src/core/geometry/qgsgeometry.h

```diff
@@ -561,6 +561,13 @@ class CORE_EXPORT QgsGeometry
     QgsGeometry orientedMinimumBoundingBox( double &area SIP_OUT, double &angle SIP_OUT, double &width SIP_OUT, double &height SIP_OUT ) const;
 
     /**
+     * Returns the minimal enclosing circle for the geometry.
+     * The center and the radius of the minimal enclosing circle will also be returned.
+     * \since QGIS 3.0
```

Comment:
Can you add an explanation of the "segments" argument here too?

Reply:
done